packages/sources/openexchangerates-test/src/index.ts

```diff
@@ -0,0 +1,26 @@
+import { expose, ServerInstance } from '@chainlink/external-adapter-framework'
+import { PriceAdapter } from '@chainlink/external-adapter-framework/adapter'
+import { customSettings } from './config'
+import includes from './config/includes.json'
+import { forex } from './endpoint'
+
+export const adapter = new PriceAdapter({
+  defaultEndpoint: forex.name,
+  name: 'OPENEXCHANGERATES',
+  customSettings,
+  endpoints: [forex],
+  includes,
+  rateLimiting: {
+    tiers: {
+      developer: {
+        rateLimit1h: 13.69,
+        note: 'only mentions monthly limits',
+      },
+      enterprise: {
+        rateLimit1h: 136.9,
+      },
```

Comment:
Even though the fw doesn't support -1 like v2, we should still add the unlimited tier. We would want to set a reasonable limit though so something like
```
rateLimit1s: 100,
rateLimit1m: 6000,
note: 'Considered unlimited tier, but setting reasonable limits',
```